From c4f9b7a38c6f4a9400c9df7e0071c7b6682dd17c Mon Sep 17 00:00:00 2001
From: Duc Dang <dhdang@apm.com>
Date: Wed, 14 Oct 2015 13:44:17 -0700
Subject: [PATCH 08/17] arm64: dts: Add USB nodes for APM X-Gene v2 platforms

This patch adds USB node into APM X-Gene v2 device tree to enable
USB support for X-Gene v2 platforms.

Signed-off-by: Duc Dang <dhdang@apm.com>
---
 arch/arm64/boot/dts/apm/apm-shadowcat.dtsi | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm64/boot/dts/apm/apm-shadowcat.dtsi b/arch/arm64/boot/dts/apm/apm-shadowcat.dtsi
index d50e19f..e7e6b4d 100644
--- a/arch/arm64/boot/dts/apm/apm-shadowcat.dtsi
+++ b/arch/arm64/boot/dts/apm/apm-shadowcat.dtsi
@@ -325,6 +325,15 @@
 			interrupts = <0x0 0x4c 0x4>;
 		};
 
+		usb0: dwusb@19000000 {
+			status = "disabled";
+			compatible = "snps,dwc3";
+			reg =  <0x0 0x19000000 0x0 0x100000>;
+			interrupts = <0x0 0x5d 0x4>;
+			dma-coherent;
+			dr_mode = "host";
+		};
+
 		pcie0: pcie@1f2b0000 {
 			status = "disabled";
 			device_type = "pci";
-- 
2.6.2

